src/vm/tests/costs.rs

```diff
@@ -49,11 +49,11 @@ fn get_simple_test(function: &NativeFunctions) -> &'static str {
         AsMaxLen => "(as-max-len? list-bar u3)",
         Len => "(len list-bar)",
         ListCons => "(list 1 2 3 4)",
-        FetchEntry => "(map-get? map-foo ((a 1)))",
-        FetchContractEntry => "(contract-map-get? .contract-other map-foo ((a 1)))",
-        SetEntry => "(map-set map-foo ((a 1)) ((b 2)))",
-        InsertEntry => "(map-insert map-foo ((a 2)) ((b 2)))",
-        DeleteEntry => "(map-delete map-foo ((a 1)))",
+        FetchEntry => "(map-get? map-foo {a 1})",
```

Comment:
ha, so `{a 1}` is a legit syntax?
we saw how confusing having 2 syntax could be (implicit vs explicit), is this a caveat we should tackle with this improvement?

Reply:
It's not different but the same syntax. The colons are purely decorative syntactical saccharine, insignificant for the semantics but (hopefully) helpful for developers familiar with languages where the colon is necessary for the parser to separate labels from values. 

This is thus much like the optional [trailing comma in Rust](https://github.com/rust-lang/rust-guidelines/issues/21), which now also is allowed in the Tuple Literals:

    {id: id,}

Reply:
Sounds good to me!